app/ts/components/pages/Home.tsx

```diff
@@ -188,13 +189,35 @@ function SimulationResults(param: SimulationStateParam) {
 				simulationAndVisualisationResults = { param.simulationAndVisualisationResults }
 				currentBlockNumber = { param.currentBlockNumber }
 				renameAddressCallBack = { param.renameAddressCallBack }
-				isConnected = { param.isConnected }
+				rpcConnectionStatus = { param.rpcConnectionStatus }
 			/>
 		}
 		<div class = 'content' style = 'height: 0.1px'/>
 	</div>
 }
 
+type NetworkErrorParams = {
+	rpcConnectionStatus: RpcConnectionStatus
+}
+
+export function NetworkErrors({ rpcConnectionStatus } : NetworkErrorParams) {
+	if (rpcConnectionStatus === undefined) return <></>
+	const priorDate = new Date(rpcConnectionStatus.lastConnnectionAttempt.getTime() + TIME_BETWEEN_BLOCKS * 1000)
+	return <>
+		{ rpcConnectionStatus.isConnected === false ?
+			<div style = 'margin: 10px; background-color: var(--bg-color);'>
+				<Error warning = { true } text = { <>Unable to connect to { rpcConnectionStatus.rpcNetwork.name }. Retrying in <SomeTimeAgo priorTimestamp = { priorDate } countBackwards = { true }/>.</> }/>
+			</div>
+		: <></> }
+
+		{ rpcConnectionStatus.latestBlock !== undefined && noNewBlockForOverTwoMins(rpcConnectionStatus) ?
+			<div style = 'margin: 10px; background-color: var(--bg-color);'>
+				<Error warning = { true } text = { <>The connected RPC ({ rpcConnectionStatus.rpcNetwork.name }) seem to be stuck at block { rpcConnectionStatus.latestBlock.number }. Retrying in <SomeTimeAgo priorTimestamp = { priorDate } countBackwards = { true }/>.</> }/>
+			</div>
```

Comment:
We should include `rpcConnectionStatus.latestBlock.timestamp` somewhere in the warning, so the user knows just how old the block is.  In fact, I would remove the number and replace it with timestamp I think.

Reply:
![image](https://github.com/DarkFlorist/TheInterceptor/assets/13102010/3207e01c-8792-4ebc-950e-19a809eac1a6)
I feel the block number is useful too, as if fork happens and somene forgots to update, the block number would somewhat explain why

Reply:
Recommend changing "occured on" to "produced at" for a bit more clarity.